calculator.js

```diff
@@ -23,6 +13,73 @@ const calculateUserInput = function (error, promptInput) {
 //   operation: 'add',
 // });
 
+function getValidOperator(operator) {
+const operators = ['+', 'add', '-', 'subtract', '*', 'multiply', '/', 'divide']; 
+
+  if (operators.includes(operator)) {
+    return operator;
+  }
+  return undefined;
+}
+
+function getValidNumber(number) {
+    return parseInt(number);
+}
+
+const calculateUserInput = function (error, promptInput) {
+  // console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
+
+  let operator = getValidOperator(promptInput.operation);
+  if (operator == null) {
+    console.log(`unsupported operator`);
+    return;
+  }
+
+
+  let num1 = getValidNumber(promptInput.num1);
+  if (isNaN(num1)) {
+    console.log(`${promptInput.num1} is not a number.`);
+    return;
+  }
+
+
+  let num2 = getValidNumber(promptInput.num2);
+  if (isNaN(num2)) {
+    console.log(`${promptInput.num2} is not a number.`);
+    return;
+  }
+
+  switch (operator) {
+    case '+':
+    case 'add':
+      console.log(`${num1} + ${num2} = ${num1 + num2}`);
+    break;
```

Comment:
`break` should be part of the `case` block:

```suggestion
      break;
```